pkgs/top-level/all-packages.nix

```diff
@@ -26506,6 +26506,7 @@ with pkgs;
   prometheus-fastly-exporter = callPackage ../servers/monitoring/prometheus/fastly-exporter.nix { };
   prometheus-flow-exporter = callPackage ../servers/monitoring/prometheus/flow-exporter.nix { };
   prometheus-fritzbox-exporter = callPackage ../servers/monitoring/prometheus/fritzbox-exporter.nix { };
+  prometheus-frr-exporter = callPackage ../by-name/fr/frr-exporter/package.nix { };
```

Comment:
It can be built without adding prometheus-frr-exporter to pkgs/top-level/all-packages.nix because you have set the derivation file in pkgs/byname.

ref: https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
What about the apparent convention (couldn't find it documented) that all exporters for prometheus should start with `prometheus-[..]`? Should I rename the package and put it into the `by-name` path instead of the remapping here?

Reply:
Moving the package into by-name/pr/prometheus-frr-exporter makes sense to me!